edx_notifications/server/web/static/edx_notifications/js/test/specs/counter_icon_view_spec.js

```diff
@@ -0,0 +1,30 @@
+describe("CounterIconView", function(){
```

Comment:
This will need to be updated to use RequireJS with https://github.com/edx/edx-notifications/pull/16.